desktop/cmp/button/Button.js

```diff
@@ -26,26 +29,29 @@ export class Button extends Component {
         onClick: PT.func,
         style: PT.object,
         text: PT.string,
-        title: PT.string
+        title: PT.string,
+        autoFocus: PT.bool
     };
 
     baseClassName = 'xh-button';
 
     render() {
-        const {icon, text, onClick, minimal = true, style, ...rest} = this.getNonLayoutProps();
+        const {icon, text, onClick, minimal = true, style, autoFocus, ...rest} = this.getNonLayoutProps(),
+            autoFocusClassName = autoFocus ? ' xh-button--autofocus-enabled' : '';
         return bpButton({
             icon,
             minimal,
             onClick,
             text,
+            autoFocus,
 
             style: {
                 ...style,
                 ...this.getLayoutProps()
             },
 
             ...rest,
-            className: this.getClassName(minimal ? 'xh-button--minimal' : '')
+            className: this.getClassName(minimal ? `xh-button--minimal${autoFocusClassName}` : autoFocusClassName)
```

Comment:
`getClassName` is setup to accept an arbitrary number of args, some of which can be empty/null, and will call the `classNames` util to concat them safely into a string.  So no need to concat strings here.

Reply:
Cleaned up.